packages/hero/index.js

```diff
@@ -40,7 +40,7 @@ function Hero({ data, centered, gaPrefix }) {
       )}
     >
       {backgroundImage && <Image className="bg" {...backgroundImage} />}
-      <div className="g-container">
+      <div className="g-grid-container">
```

Comment:
Ran a global find-and-replace of `g-container` for `g-grid-container`. This was a holdout.